bootstrap/src/main/java/org/jboss/windup/bootstrap/commands/windup/RunWindupCommand.java

```diff
@@ -129,7 +131,7 @@ private void runWindup(List<String> arguments)
             }
             else if (Boolean.class.isAssignableFrom(option.getType()))
             {
-                optionValues.put(option.getName(), true);
+                optionValues.put(option.getName(), !"false".equalsIgnoreCase(argument));
```

Comment:
If I understand this line correctly, it will introduce a significant behavior change. Previously specifying something like --sourceMode would automatically set that attribute to true.